assets/externalized/strategic-map-movement-costs.json

```diff
@@ -0,0 +1,115 @@
+	/* IH */ [ "G", "G", "R", "D", "P", "R", "P", "P", "A", "A", "P", "S", "X", "X", "P", "P" ],
+	/* IJ */ [ "G", "G", "R", "W", "D", "R", "P", "A", "A", "A", "P", "P", "S", "R", "P", "E" ],
+	/* KJ */ [ "G", "R", "P", "W", "W", "R", "A", "A", "R", "P", "P", "P", "P", "R", "P", "E" ],
+	/* KL */ [ "G", "R", "W", "W", "D", "R", "A", "A", "A", "R", "R", "R", "P", "P", "P", "E" ],
+	/* ML */ [ "G", "R", "W", "P", "P", "R", "P", "A", "P", "R", "G", "G", "P", "P", "E", "E" ],
+	/* MN */ [ "G", "G", "R", "P", "R", "P", "P", "P", "P", "R", "G", "G", "E", "E", "E", "E" ],
+	/* ON */ [ "G", "G", "X", "X", "G", "G", "G", "P", "P", "G", "G", "G", "E", "E", "E", "E" ],
+	/* OP */ [ "G", "G", "X", "G", "G", "G", "G", "G", "G", "G", "G", "G", "G", "E", "E", "E" ],
+	/*  P */ [ "E", "E", "E", "E", "E", "E", "E", "E", "E", "E", "E", "E", "E", "E", "E", "E" ]  // always E
+	],
+	// Traversiblity of going through the sector without entering it
+	"traverseThrough": [
+	/*         1    2     3     4     5     6     7     8     9     10    11    12    13    14    15    16 */
+	/* A */ [ "T", "X",  "P",  "E",  "E",  "H",  "H",  "H",  "X",  "X",  "F",  "D",  "S",  "P",  "D",  "E"  ],
+	/* B */ [ "T", "X",  "S",  "S",  "F",  "F",  "P",  "F",  "PR", "SR", "SR", "FR", "X",  "WT", "W",  "D"  ],
+	/* C */ [ "T", "TR", "PR", "S",  "X",  "X",  "PR", "SR", "PR", "S",  "D",  "S",  "X",  "WT", "W",  "W"  ],
+	/* D */ [ "G", "X",  "PR", "H",  "X",  "F",  "DR", "D",  "PR", "P",  "S",  "W",  "X",  "WT", "X",  "D"  ],
+	/* E */ [ "G", "T",  "PR", "H",  "H",  "P",  "FR", "F",  "FR", "W",  "S",  "S",  "WT", "W",  "DR", "E"  ],
+	/* F */ [ "G", "C",  "PR", "H",  "H",  "H",  "HR", "X",  "X",  "F",  "P",  "SR", "S",  "PR", "DR", "E"  ],
+	/* G */ [ "X", "X",  "CR", "HR", "HR", "HR", "FR", "X",  "X",  "AR", "AR", "PR", "P",  "PR", "D",  "W"  ],
+	/* H */ [ "X", "X",  "X",  "D",  "H",  "HR", "P",  "X",  "P",  "A",  "A",  "PR", "X",  "X",  "W",  "S"  ],
+	/* I */ [ "G", "G",  "CR", "D",  "S",  "X",  "S",  "X",  "A",  "A",  "P",  "P",  "X",  "X",  "S",  "S"  ],
+	/* J */ [ "G", "CR", "CR", "W",  "W",  "SR", "S",  "A",  "X",  "A",  "A",  "S",  "S",  "FR", "S",  "E"  ],
+	/* K */ [ "G", "CR", "W",  "X",  "W",  "PR", "AR", "AR", "AR", "PR", "PR", "PR", "SR", "SR", "D",  "E"  ],
+	/* L */ [ "C", "CR", "W",  "W",  "D",  "PR", "P",  "A",  "P",  "SR", "X",  "X",  "C",  "W",  "D",  "E"  ],
+	/* M */ [ "G", "CR", "WR", "F",  "DR", "DR", "P",  "P",  "P",  "TR", "G",  "G",  "S",  "W",  "E",  "E"  ],
+	/* N */ [ "G", "G",  "X",  "X",  "X",  "TR", "CR", "CR", "TR", "TR", "G",  "G",  "E",  "E",  "E",  "E"  ],
+	/* O */ [ "G", "G",  "X",  "X",  "G",  "G",  "G",  "T",  "T",  "G",  "G",  "G",  "G",  "E",  "E",  "E"  ],
+	/* P */ [ "G", "G",  "X",  "G",  "G",  "G",  "G",  "G",  "G",  "G",  "G",  "G",  "G",  "E",  "E",  "E"  ]
+	],
+	// It is supposed to mean how travelled a sector is. The higher the travel rating, the more people go near it.
```

Comment:
IDK if I would even include this if it is not used.

Reply:
I don't know either. 

Save games still expects the Travel Rating value.  We can either leave it here in JSON and never touch it again (and in case somehow we want to reference or use it in the future), or we can hard-code the values all to zero.

Reply:
Let's keep it.